### PR TITLE
added Pax Viking

### PR DESCRIPTION
**Pull request by Gimleux**
Submitting this PR means you have:

- [x] searched the [readme][readme-url] file and searched the [open pull requests][pr-url] (no duplicates please)
- [x] and will follow [the desired format][format-url]
- [x] used image links that resolve correctly, are appropriately sized, and are served over HTTPS

Once your PR is created, it will trigger a build a GitHub Action (Awesome-Lint). This includes a format linting process, so if there are any issues with the build, please address them.

Do you have further questions? Check the [contribution guidelines][contributing-url] or [open an issue][new-issue-url].

When you're ready, check off the above acknowledgements and continue with your Pull Request... and thanks!

[pr-url]: https://github.com/edm00se/awesome-board-games/pulls
[readme-url]: https://github.com/edm00se/awesome-board-games/edit/main/readme.md
[format-url]: https://github.com/edm00se/awesome-board-games/blob/main/formatting.md
[contributing-url]: https://github.com/edm00se/awesome-board-games/blob/main/contributing.md
[new-issue-url]: https://github.com/edm00se/awesome-board-games/issues/new

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -658,6 +658,16 @@
...
 
+### [Pax Viking](https://boardgamegeek.com/boardgame/303954/pax-viking)
+
+> Pax Viking is a card driven game where you compete over cards in a common market in combination with strategic movement of longships on the map. You can play cards you buy on map locations you control through your longships; create trading posts, claim benefits from events and more. You can also play cards on your jarlboard, adding abilities to your player through friends, allies and gods. Through your actions you collect followers to give you power and influence for your final bid: to be the first king of Sweden.
+
+![Pax Viking image](https://spieltraum-shop.de/media/image/a3/40/05/show-id-1619527818652747_200x200.jpg)
+
+| Players | Min. Age |  Time   |
+| ------: | -------: | ------: |
+|   (1 -) 3 - 6 |       12 | 20m * number of players |
```
Comment (edm00se) [inline readme.md:669]: Please update the time to reflect the vendor’s time range, as well as the player number count for the same reasons as your other PR.

**Review by edm00se — CHANGES_REQUESTED** [review]
Just a couple of tweaks and this one will be ready as well.

File: readme.md (hunk 1 above)
Comment (Gimleux) [reply readme.md]: The time actually reflects (or seemingly reflected) the vendor's time range and is stated like this on the box. Don't know why they changed it online as the more static frame is also more unrealistic.

Swapped it in the PR.

Comment (edm00se) [reply readme.md]: Interesting, I would guess “to prevent confusion” in spite of a more accurate value. In any case, thank you for updating.

**Review by edm00se — APPROVED** [review]
Thanks for contributing!

**Comment by edm00se** [comment]
ref: #230

**Pull request merged**